Mathlib/LinearAlgebra/Matrix/NonsingularInverse.lean

```diff
@@ -737,6 +737,16 @@ theorem inv_kronecker [Fintype m] [DecidableEq m]
   · apply inv_eq_right_inv
     rw [← mul_kronecker_mul, ← one_kronecker_one, mul_nonsing_inv _ hA, mul_nonsing_inv _ hB]
 
+lemma _root_.LinearEquiv.toMatrix_inv {M : Type*} [AddCommMonoid M] [Module α M]
+    (v w : Basis n α M) (f : M ≃ₗ[α] M) :
+    (f.toMatrix v w)⁻¹ = f⁻¹.toMatrix w v :=
```

Comment:
I think this and the one below hold more generally for `Ring.inverse` and `LinearMap`?

Reply:
Can `(LinearMap.toMatrix w w) (f ∘ₗ Ring.inverse f) = 1` be proved? It can happen that the LHS is zero, right?

Reply:
Correct, you should check if both sides are zero before rearranging.